src/app/public/components/search-box/search-box.component.ts

```diff
@@ -1,4 +1,15 @@
-import { Component, OnInit, EventEmitter, Output, ElementRef, ViewChild, AfterViewInit, HostBinding, Inject } from '@angular/core';
+/* @format */
+import {
+  Component,
+  OnInit,
+  EventEmitter,
+  Output,
+  ElementRef,
+  ViewChild,
+  AfterViewInit,
+  HostBinding,
+  Input,
```

Comment:
I think I removed Inject from the import list since it wasn't being used.  Previous the SearchBoxComponent did not have any Inputs so I did have to add Input to the import list.